build/target-repository/docs/rector_rules_overview.md

```diff
@@ -2073,7 +2073,7 @@ use Rector\CodingStyle\Rector\ClassMethod\OrderAttributesRector;
 use Rector\Config\RectorConfig;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->ruleWithConfiguration(OrderAttributesRector::class, ['First', 'Second']);
+    $rectorConfig->ruleWithConfiguration(OrderAttributesRector::class, [OrderAttributesRector::ALPHABETICALLY]);
 };
 ```
 
```

Comment:
The original test should be kept, as that's current behavior.

Could you create a separate test with no configuration?

Reply:
I don't really know how to keep and previous behavior and the new one.

If we want to keep the previous behavior I will have to create another option:
```php
public const SPECIFIC = 'specific';
```
and use it it like this ?
```php
$rectorConfig->ruleWithConfiguration(OrderAttributesRector::class, [OrderAttributesRector::SPECIFIC => ['First', 'Second']]);
```

Reply:
Magic methods removal static analysis and make more mess. Better use explicit method names.

Reply:
Sorry I didn't anderstand what you meant.

> Better use explicit method names

But you also write:
> This behavior should be kept.
For:
```php
$currentAttrGroups = $this->{$this->option}($originalAttrGroups);
```